### DISCUSSION TITLE
Errors In Worker Log - Actions work fine

### DISCUSSION THREAD

**Discussion by cryoaura** [apioo/fusio#549] [General]
I seem to be getting a lot of errors with no context in them everytime an action runs - whether by cron, execute in designer, or through an operation call. The actions complete just fine and no errors are shown on screen when I get the return in designer manually running the action.

FastCGI sent in stderr: "PHP message: [Time] Worker.ERROR: Execute action [Action Name] [] []" while reading response header from upstream, client

The [Time] and [Action Name] do actually show the correct information. This error seems to pop up on every single PHP worker action. Normally I wouldnt care, but it's clogging up the log file for the worker and also just bothers me. Am I doing something incorrect in the worker?

I'm following the format that's an example on https://github.com/apioo/fusio-worker-php.

**Comment by chriskapp** [comment]
Hi @cryoaura this is actually no error but only a log message by the worker, so you can ignore this message. FYI we are currently re-designing the worker system for the next Fusio 5.0 release, with this step we move away from Apache Thrift and implement the worker as simple REST API, with this step we will also remove those messages.
